typescript/api/services/SvgSanitizerService.ts

```diff
@@ -0,0 +1,425 @@
+import { PopulateExportedMethods, Services as coreServices } from '@researchdatabox/redbox-core-types';
```

Comment:
There's a lot of assumed knowledge in the sanitisation methods in this service.
Where did the info about how & what to sanitise come from?

There need to be comments with references and other notes about why and how things need to be done, and how we have assured ourselves that the approaches work.

Reply:
Could we use existing libraries to do more of this?
The main reference for sanitising HTML and other formats, that I know of, is OWASP: https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#html-sanitization